docs/source/release/v6.5.0/index.rst

```diff
@@ -16,7 +16,25 @@ Mantid 6.5.0 Release Notes
 
```

Comment:
Can we remove the `.. warning::` above this line. We're getting close enough to release now that it's not worth us accidentally leaving this in.